<?xml version="1.0"?>
<launch>

  <arg name="node_name"     default="node_01" />

  <!--***********************************************************************-->
  <!-- Hi-Ros Orientation Skeleton Tracker Node: Parameters override and Run -->
  <!--***********************************************************************-->
  <include file="$(find hiros_skeleton_tracker)/launch/hiros_orientation_skeleton_tracker_default.launch">
    <arg name="node_required"                        value="true"                                           />
    <arg name="node_name"                            value="$(arg node_name)"                               />

    <arg name="in_skeleton_group_topics"             value="[/topic1, /topic2]"                             />
    <arg name="out_msg_topic_name"                   value="orientation_skeleton_group"                     />

    <arg name="fixed_delay"                          value="0.1"                                            />
    <arg name="min_orientations"                     value="7"                                              />
    <arg name="min_distance"                         value="0.5"                                            />
    <arg name="max_distance"                         value="100"                                            />
    <arg name="max_delta_t"                          value="0.05"                                           />
    <arg name="use_orientations"                     value="true"                                           />
    <arg name="use_velocities"                       value="true"                                           />
    <arg name="velocity_weight"                      value="0.05"                                           />
    <arg name="weight_distances_by_confidences"      value="false"                                          />
    <arg name="weight_distances_by_velocities"       value="false"                                          />
  </include>

</launch>
